### PR TITLE
Migrate fedora-server installclass into the anaconda source repositor…

### PR DESCRIPTION
**Pull request by rvykydal**
…y (#1466967)

We can remove it from fedora-productimg-server then.

### CODE REVIEW

File: pyanaconda/installclasses/fedora-server.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+#
+# fedora.py
```
Comment (poncovka) [inline pyanaconda/installclasses/fedora-server.py]: This is `fedora-server.py`.

File: pyanaconda/installclasses/fedora-server.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
+#
```
Comment (poncovka) [inline pyanaconda/installclasses/fedora-server.py]: Could be the title a little shorter in the commit message? :-)

File: pyanaconda/installclasses/fedora-server.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+# fedora.py
+#
+# Copyright (C) 2017  Red Hat, Inc.  All rights reserved.
```
Comment (sgallagher) [inline pyanaconda/installclasses/fedora-server.py]: This is actually Copyright (C) Stephen Gallagher at the moment (I don't assign copyright to Red Hat by default).

If there are reasons that this needs to be assigned, please request it. In general, I prefer to retain my right to have a say in relicensing decisions.

**Comment by rvykydal** [comment]
jenkins, test this please

**Comment by rvykydal** [comment]
jenkins, test this please

**Comment by rvykydal** [comment]
jenkins, test this please

**Comment by jkonecny12** [comment]
jenkins, test this please

**Comment by rvykydal** [comment]
jenkins, test this please

**Comment by jkonecny12** [comment]
Fix these errors and I guess it should be ok.

```
************* Module ../pyanaconda/kickstart.py
W9901(found-percent-in-_):1177,44: MountData.execute: Found % in a call to a _() method
W9901(found-percent-in-_):1182,28: MountData.execute: Found % in a call to a _() method
W9901(found-percent-in-_):1189,52: MountData.execute: Found % in a call to a _() method
```

**Comment by rvykydal** [comment]
jenkins, test this please

**Pull request merged**